contrib/win32/install/build/README.md

```diff
@@ -0,0 +1,43 @@
+# Windows OpenSSH - Build Scripts
+
+## `build-openssh.ps1`
```

Comment:
We already have [openssh-build.ps1](https://github.com/PowerShell/openssh-portable/blob/latestw_all/contrib/win32/openssh/OpenSSH-build.ps1) which does the build and package.

Please modify this.